<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wVerifyMap</class>
 <widget class="QMainWindow" name="wVerifyMap">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">wVerifyMap</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/rec/data/icons/wVerifyMap.svg</normaloff>:/rec/data/icons/wVerifyMap.svg</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="cobxMapName">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="readOnly" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReloadMaps">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="view-refresh"/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnVerificationPreferences">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="document-properties"/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnStartVerifying">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="text">
         <string>Start!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="twgVerfying">
      <property name="toolTip">
       <string/>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::TabPosition::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::TabShape::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::TextElideMode::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tabOverview">
       <property name="toolTip">
        <string>Overview and statistics about the map</string>
       </property>
       <attribute name="title">
        <string>Overview</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="wdgOverviewTile" name="wdgTilesOverview" native="true"/>
          </item>
          <item>
           <widget class="wdgOverviewTile" name="wdgSceneryobjectsOverview" native="true"/>
          </item>
          <item>
           <widget class="wdgOverviewTile" name="wdgSplinesOverview" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>66</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="wdgOverviewTile" name="wdgVehiclesOverview" native="true"/>
          </item>
          <item>
           <widget class="wdgOverviewTile" name="wdgHumansOverview" native="true"/>
          </item>
          <item>
           <widget class="wdgOverviewTile" name="wdgTexturesOverview" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lblPicture">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>280</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabTiles">
       <attribute name="title">
        <string>Tiles</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="wdgTab" name="wdgTiles" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSceneryobjects">
       <property name="toolTip">
        <string>Sceneryobjects on the map</string>
       </property>
       <attribute name="title">
        <string>Sceneryobjects</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="wdgTab" name="wdgSceneryobjects" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSplines">
       <property name="toolTip">
        <string>Splines on the map</string>
       </property>
       <attribute name="title">
        <string>Splines</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="wdgTab" name="wdgSplines" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabVehicles">
       <property name="toolTip">
        <string>AI vehicles on the map</string>
       </property>
       <attribute name="title">
        <string>Vehicles</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="wdgTab" name="wdgVehicles" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabHumans">
       <property name="toolTip">
        <string>Humans on the map</string>
       </property>
       <attribute name="title">
        <string>Humans</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="wdgTab" name="wdgHumans" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabTextures">
       <attribute name="title">
        <string>Textures</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="wdgTab" name="wdgTextures" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="pgbProgress">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::Direction::TopToBottom</enum>
      </property>
      <property name="format">
       <string notr="true">%p%</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="actionStartVerifying"/>
    <addaction name="separator"/>
    <addaction name="actionBackToHome"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuSpecials">
    <property name="title">
     <string>Specials</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionSendFeedback"/>
   </widget>
   <addaction name="menuMap"/>
   <addaction name="menuSpecials"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Q</string>
   </property>
  </action>
  <action name="actionStartVerifying">
   <property name="icon">
    <iconset theme="edit-find"/>
   </property>
   <property name="text">
    <string>Start verifying</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+R</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset theme="document-properties"/>
   </property>
   <property name="text">
    <string>Preferences...</string>
   </property>
   <property name="toolTip">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+E</string>
   </property>
  </action>
  <action name="actionSendFeedback">
   <property name="icon">
    <iconset theme="dialog-question"/>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string notr="true">F1</string>
   </property>
  </action>
  <action name="actionBackToHome">
   <property name="icon">
    <iconset theme="go-home"/>
   </property>
   <property name="text">
    <string>Back to home</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+W</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wdgOverviewTile</class>
   <extends>QWidget</extends>
   <header>OTModules/wVerifyMap/Widgets/wdgoverviewtile.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>wdgTab</class>
   <extends>QWidget</extends>
   <header>OTModules/wVerifyMap/Widgets/wdgtab.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
